servicebus/spring-cloud-azure-starter-integration-servicebus/multiple-namespaces/terraform/main.tf

```diff
@@ -0,0 +1,120 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 2.75"
+    }
+    azurecaf = {
```

Comment:
what's this?

Reply:
this is used for [azurecaf_name](https://registry.terraform.io/providers/aztfmod/azurecaf/latest/docs/resources/azurecaf_name), to generate Azure Resource name String